backend/typescript/rest/campRoutes.ts

```diff
@@ -26,9 +27,10 @@ const fileStorageService: IFileStorageService = new FileStorageService(
 
 const campService: ICampService = new CampService(fileStorageService);
 /* Get all camps */
-campRouter.get("/", async (req, res) => {
+campRouter.get("/", getCampDtoValidator, async (req, res) => {
   try {
-    const camps = await campService.getCamps();
+    const { campYear } = req.query;
+    const camps = await campService.getCamps(parseInt(campYear as string, 10));
```

Comment:
I tested this so I know it works but one of the concerns I had was if we don't pass in campYear then how does this parseInt ?

Reply:
I think it would just call the getCamps function with no value for the `year` argument. Since we will make the filter on the camps after calling all camps if a `year` exists, it just returns all the camps queried as required